Dockerfile.assisted-installer-build

```diff
@@ -2,7 +2,7 @@ FROM registry.ci.openshift.org/openshift/release:golang-1.17
 ENV GO111MODULE=on
 ENV GOFLAGS=""
 
-RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.25.0
+COPY --from=quay.io/app-sre/golangci-lint:v1.25.0 /usr/bin/golangci-lint /usr/bin/golangci-lint
```

Comment:
Doesn't need chmod?

Reply:
(the original install.sh script ran `install` (`man install`) which sets chmod on the copied bihnary)

Reply:
Seems like it copies it with the right permissions:
```
$ ls -la /usr/bin/golangci-lint
-rwxrwxr-x 1 root root 21929984 Apr 22  2020 /usr/bin/golangci-lint
```
(from inside the assisted-installer-build container)

Reply:
cool